README.md

```diff
@@ -57,58 +54,76 @@ snap connect wifi-connect:network-setup-control core:network-setup-control
 
 Note: wifi-ap and network-manager interfaces auto-connect.
 
-## If you configured wifi in console-conf
+# Set NetWorkManager to control all networking
+
+Note: This is a temporary manual step before network-manager snap provides a config option for this.
+
+Note: Depending on your environment, after this you may need to use a new IP address to connect to the device.
 
-The netplan file needs to be modified to set wlan0 as managed by network manager:
+1. Backup the existing /etc/netplan/00-snapd-config.yaml file 
 
 ```bash
-sudo wifi-connect.netplan
+sudo mv /etc/netplan/00-snapd-config.yaml ~/
+```
+
+1. Create a new netplan config file named /etc/netplan/00-default-nm-renderer.yaml:
+
+``bash
+sudo vi /etc/netplan/00-default-nm-renderer.yaml
+
 ```
 
-This command modifies the default netplan file. You can either run `sudo netplan generate && netplan apply` or reboot (the next step).
+Add the following two lines:
+
+```bash
+network:
+    renderer: NetworkManager
+```
 
 ## Reboot
 
-The content sharing interface has a known issue. Until that is resolved, you need to restart the system at this point.
+Rebooting addresses a content sharing interface issue. 
+
+Rebooting also consolidates all networking into NetworkManager.
 
 ## Optionally configure wifi-ap SSID/passphrase
 
 If you skip these steps, the wifi-AP put up by the device has an SSID of "Ubuntu" and is unsecure (with no passphrase). 
 
-1. Stop the daemon
+1. Stop wifi-connect
 
 ```bash
-sudo systemctl stop snap.wifi-connect.daemon.service
+sudo  wifi-connect stop
 ```
 
-1. Bring the AP down:
+1. Set the wlan0 interface to be unmanaged by NetworkManager
 
 ```bash
-sudo wifi-connect.wifi-ap -ap-off
+nmcli set wlan0 managed n
```

Comment:
Is this command correct? I'm getting this https://pastebin.canonical.com/188510/